examples/chip-tool/templates/partials/test_cluster.zapt

```diff
@@ -197,6 +197,15 @@ class {{filename}}: public TestCommand
             return;
           }
         {{/if}}
+
+        {{#if expectedConstraints.notValue}}
+          if ({{asLowerCamelCase name}} == {{expectedConstraints.notValue}})
+          {
+            ChipLogError(chipTool, "Error: {{asLowerCamelCase name}} is equal to an unexpected value. Not value is {{expectedConstraints.notValue}} but got '%d'", {{asLowerCamelCase name}});
```

Comment:
text feels odd: "Not value is 123, but got '123'". Maybe something like this sounds better:

"Value was not expected to be %d (due to notValue constraint)"